packages/web/src/common/values/token-constant.ts

```diff
@@ -11,6 +11,7 @@ export const GNS_TOKEN: TokenModel = {
   logoURI:
     "https://raw.githubusercontent.com/onbloc/gno-token-resource/main/grc20/images/gno_land_r_gns.svg",
   priceID: "gno.land/r/demo/gns",
```

Comment:
Please remove this item to avoid confusion.

Reply:
i cannot remove until api update, this field exist in current model